composer.json

```diff
@@ -10,8 +10,6 @@
     "type": "shopware-platform-plugin",
     "license": "MIT",
     "require": {
```

Comment:
the plugin will use the core shopware libraries from the shopware installation, no need to require them here

Reply:
reverted.